### DISCUSSION TITLE
为啥我Java17的1.19.3装不了forge最新版

### DISCUSSION THREAD

**Discussion by Leo-gezi** [Meloong-Git/PCL#1118] [提问] [answered]
_**还有不要点击这个按钮其实应该就是 Jeff吧，我盆友发过眼睛闪瞎了（滑稽）**_

**Comment by WForst-Breeze** [comment, marked as answer]
请提供启动器日志等相关详细信息以让我们更快地确认问题所在，这里给出以下两种**可能的解决方案**：
1. 检查您的网络，如有必要请使用 VPN
2. Forge 安装时间较长，耐心等待即可~
